coalib/coala_main.py

```diff
@@ -52,6 +54,10 @@ def run_coala(log_printer=None,
     :return:                        A dictionary containing a list of results
                                     for all analyzed sections as key.
     """
+    # Configure logging
+    with open('logging_config.json') as fd:
+        logging.config.dictConfig(json.load(fd))
```

Comment:
btw. you want to load in a global config or should the user write it? The one below looks global but here youre loading it from current directory